GETTING_STARTED.md

```diff
@@ -46,6 +46,18 @@ Add HammerJS to your application via [npm](https://www.npmjs.com/package/hammerj
 (such as the [Google CDN](https://developers.google.com/speed/libraries/#hammerjs)), or served 
 directly from your app.
 
+```bash
```

Comment:
This just needs a title like "If you want to include HammerJS from npm, you can install it:"